ludwig/utils/upload_utils.py

```diff
@@ -256,17 +258,32 @@ def upload(
         )
 
         # Upload all artifacts in model weights folder
-        upload_path = self.api.upload_folder(
+        commit_message_weights = f"{commit_message} (weights)" if commit_message else commit_message
```

Comment:
@sanjaydasgupta This is wonderful -- I would like to just request type-hints (we would like to move towards type hints uniformly for a better quality of code -- so this one would read `commit_message_weights: str`).  Thank you very much.